### ISSUE TITLE
inference

### ISSUE THREAD

**Issue by lalalabobobo** [ardha27/AI-Song-Cover-SOVITS#7]
May I ask if you can use the mp3 or wav file you uploaded for inference?

**Comment by aiprojectchiwa** [comment]
yes,but you can only use the .wav file for inference,for the better ,use the .wav file from Separated feature
